episodes/07-thresholding.md

```diff
@@ -348,13 +348,17 @@ plt.xlim(0, 1.0)
 ![](fig/maize-root-cluster-histogram.png){alt='Grayscale histogram of the maize root image'}
 
 The histogram has a significant peak around 0.2, and a second,
-smaller peak very near 1.0.
+shallower peak near 0.6.
 Thus, this image is a good candidate for thresholding with Otsu's method.
 The mathematical details of how this works are complicated (see
 [the scikit-image documentation](https://scikit-image.org/docs/dev/api/skimage.filters.html#threshold-otsu)
 if you are interested),
 but the outcome is that Otsu's method finds a threshold value between
 the two peaks of a grayscale histogram.
+Note there is a third peak very close to 1.0 which corresponds to the white label and disk on the 
+left hand side of the image. It would be a good idea to remove these white pixels 
+before calculating the Otsu threshold. We will discuss how to do this later in the episode but for 
+now we will continue without removing them.
```

Comment:
While it is undoubtedly accurate, I am not sure I prefer this to the current version: as currently written, I believe the idea is for Instructors to naively proceed on the assumption that the second-largest peak is the stuff we are most interested in in the image: sort of a binary way of thinking about the image as composed of background and foreground which might match quite well the learners' misconceptions. Then later to help learners re-assess that notion and realise that further selection needs to take place to isolate only the pixels of interest.

I would favour a note added to the "Ignoring more of the image" challenge, stating that really it would have been better practice to remove these pixels too before calculating the Otsu threshold in the first place. Interested to hear what others think @datacarpentry/image-processing-curriculum-maintainers

Reply:
I agree with @tobyhodges's comment: From an educational viewpoint, I think that we want to proceed 'naively' indeed; we don't want to 'anticipate' oddities, but instead 'react' to these afterwards (let learners see them and update their mental models then).

Reply:
Great - now in a instructor note as per @uschille suggestion to avoid overloading the learners